### PR TITLE
15815 convert dashboard widgets for users/groups

### PR DESCRIPTION
**Pull request by arthanson**
### Fixes: #15815 

Convert dashboard widgets using `users.netboxuser` -> `users.user` and `users.netboxgroup` -> `users.group`

### CODE REVIEW

File: netbox/extras/migrations/0115_convert_dashboard_widgets.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+from django.db import migrations
+
+
+def update_dashboard_widgets(apps, schema_editor):
+    Dashboard = apps.get_model('extras', 'Dashboard')
+
+    for dashboard in Dashboard.objects.all():
+        for key, widget in dashboard.config.items():
+            breakpoint()
```
Comment (jeremystretch) [inline netbox/extras/migrations/0115_convert_dashboard_widgets.py:11]:
```suggestion
```

File: netbox/extras/migrations/0115_convert_dashboard_widgets.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+            if 'models' in widget['config']:
+                models = widget['config']['models']
```
Comment (jeremystretch) [inline netbox/extras/migrations/0115_convert_dashboard_widgets.py:13]:
```suggestion
            if models := widget['config'].get('models'):
```

**Review by jeremystretch — CHANGES_REQUESTED** [review]
It feels like we should also take this opportunity to improve error handling in `extras.dashboard.widgets.get_models_from_content_types()`. IMO it's reasonably to fail silently when attempting to resolve a content type that doesn't exist.

File: netbox/extras/migrations/0115_convert_dashboard_widgets.py (hunk 1 above)
Comment (arthanson) [reply netbox/extras/migrations/0115_convert_dashboard_widgets.py]: doh!

**Review by jeremystretch — APPROVED** [review]
Thanks @arthanson!

**Pull request merged**